### DISCUSSION TITLE
Process crashed Simulation failed with exit code 0xffffffffc0000135.

### DISCUSSION THREAD

**Discussion by shiyishiba** [OpenModelica/OpenModelica#12339] [Q&A]
I have imported the Modelica Buildings Library in OpenModelica and attempted to simulate an example from the thermalZone/energyPlus/examples/airHeating directory, but I encountered an error. Could professionals here provide a potential solution? Many thanks in advance！
![20240429112623](https://github.com/OpenModelica/OpenModelica/assets/125559199/65932e3a-1957-4530-b1ef-35968cd3176c)

**Comment by casella** [comment]
@shiyishiba according to [this report](https://libraries.openmodelica.org/branches/master/Buildings_10/Buildings_10.html), most of the models in that sub-package should be running fine with the latest development version of OpenModelica, including `SingleFamilyHouse.AirHeating`. We are currently working hard to fix all the remaining issues that Buildindgs has with OpenModelica and we recently fixed a few issues, please try with the latest nightly build.

**Reply by shiyishiba** [reply]
Thanks for your reply. I did use the latest version, but what was confusing was that on my two different devices, one worked fine and the other got this error.

**Comment by perost** [comment]
Does this happen only with that model, or does it happen with all models?

The c0000135 exit code is STATUS_DLL_NOT_FOUND, meaning some required DLL file couldn't be found. Unfortunately the error doesn't say which DLL, so it's hard to say if the issue is with the OM installation or the library.

**Reply by shiyishiba** [reply]
Many thanks for your response. To offer more specific details, during the simulation, the system prompts with two errors: "VCRUNTIME140D.dll not found, unable to execute code" and "ucrtbased.dll not found, unable to execute code." I would be immensely grateful if you could guide me on how to resolve these issues further. Thank you very much in advance.

**Comment by casella** [comment]
OK, this makes sense. @shiyishiba can you please open a proper ticket about that?

**Comment by adrpo** [comment]
As far as I know the mingw64 we use in OpenModelica versions 1.22 and below is based on VCRUNTIME140.dll and that dll is not shipped in Windows 11 anymore.
One could try to use the nightly build instead which uses the ucrt64 runtime and not the mingw64 one.

**Reply by casella** [reply]
Ouch. @adrpo does this mean that Win 11 support is shaky until we release 1.24.0 at the end of 2024? In that case, maybe we should consider anticipating 1.24.0 to end of July or something?

**Reply by casella** [reply]
Of course we had no problem in the [test report](https://libraries.openmodelica.org/branches/master/Buildings_10/Buildings_10.html) because that ran on Linux 😅

**Comment by adrpo** [comment]
It might also work if you install Microsoft Visual C++ 2015 Redistributable from Microsoft.
